KitabGhar/UserLibraryBookDetails.jsp

```diff
@@ -107,11 +107,11 @@
                         <form name="reviewForm" method="post" id="formRating" action="reviewSubmitServlet">
                             <br>
                             <div class="rating" style="margin-right: 100px;">
-                                <span><input type="radio" name="rating" id="str5" value="5"><label for="str5"></label></span>
-                                <span><input type="radio" name="rating" id="str4" value="4"><label for="str4"></label></span>
-                                <span><input type="radio" name="rating" id="str3" value="3"><label for="str3"></label></span>
-                                <span><input type="radio" name="rating" id="str2" value="2"><label for="str2"></label></span>
-                                <span><input type="radio" name="rating" id="str1" value="1"><label for="str1"></label></span>
+                                <span><input type="radio" name="rating" id="str5" value="5"><label for="str5"><img style="width: 25px; height: 25px" src="ProjectImages/star.png" /></label></span>
+                                <span><input type="radio" name="rating" id="str4" value="4"><label for="str4"><img style="width: 25px; height: 25px" src="ProjectImages/star.png" /></label></span>
+                                <span><input type="radio" name="rating" id="str3" value="3"><label for="str3"><img style="width: 25px; height: 25px" src="ProjectImages/star.png" /></label></span>
+                                <span><input type="radio" name="rating" id="str2" value="2"><label for="str2"><img style="width: 25px; height: 25px" src="ProjectImages/star.png" /></label></span>
+                                <span><input type="radio" name="rating" id="str1" value="1"><label for="str1"><img style="width: 25px; height: 25px" src="ProjectImages/star.png" /></label></span>
                             </div>
```

Comment:
![screenshot from 2018-10-04 11-28-37](https://user-images.githubusercontent.com/13948542/46455512-03265d80-c7c9-11e8-877f-58353c95edae.png)

*Kindly try removing the surrounding radio button default label that surrounds the star. It will be really great with only star as the label.*